runtime-testsuite/resources/org/antlr/v4/test/runtime/templates/CompositeParsers/DelegatorInvokesDelegateRuleWithArgs.stg

```diff
@@ -28,5 +28,5 @@ WS : (' '|'\n') -> skip ;
 
 slaveGrammar(grammarName) ::= <<
 parser grammar S;
-a[int x] returns [int y] : B {<write("\"S.a\"")> $y=1000;} ;
```

Comment:
:+1: That's awesome news. We can use this immediately to eliminate the need for the `more()` template, and I filed issue #931 to create new tests that ensure this basic functionality works for future targets as well.